lib/LocalEnsemble.js

```diff
@@ -198,6 +198,17 @@ if (NODEJS) {
 }
 
 
+function intersect(array1, array2) {
+    var temp;
+    if (array2.length > array1.length) {
+      temp = array2, array2 = array1, array1 = temp;
+    } // indexOf to loop over shorter
+    return array1.filter(function (e) {
+        if (array2.indexOf(e) !== -1) return true;
```

Comment:
there's no else, that's okay? why not simply `return (array2.indexOf(e) !== -1)`?